### ISSUE TITLE
lag spike and hogging cpu usage

### ISSUE THREAD

**Issue by monsef987** [TiberiumFusion/FixedSteamFriendsUI#14]
hello
after updating to patch 3.0.0 i noticed that my pc started to get slow when friend list is open and high cpu usage

![image](https://github.com/TiberiumFusion/FixedSteamFriendsUI/assets/54505057/53515427-7c96-4e26-87c1-31a1a1dbc19f)

**Comment by TiberiumFusion** [comment]
Thanks for the report.

I've checked just now and this issue also occurrs on my PC. I did not notice this before the v3.0.0 release, so that sucks.

I've also identified the source of the issue. It is caused by one of the [patches](https://github.com/TiberiumFusion/FixedSteamFriendsUI/blob/2db8650a978981090d6fd97a3301ff31c37a1547/SnapshotMaker.TsJsRewriter/Scripts/Patches/Definitions/FixBrokenInviteListInviteItems.ts) to the inner friends.js script. Changing this specific patch will resolve the issue.

I will release a hotfix soon.
